core/nodepool/config/templates/stack-template.json

```diff
@@ -352,7 +360,7 @@
                   "Effect": "Allow",
                   "Resource": "*"
                 },
-                {{if .LoadBalancer.Enabled}}
+                {{if or (.LoadBalancer.Enabled)  (.TagrgetGroup.Enabled)}}
```

Comment:
Did anyone knows whats the problem here? https://travis-ci.org/coreos/kube-aws/builds/204971257#L362